### PR TITLE
WT-8879 Set the OOO flag when the selected tombstone is globally visible

### PR DESCRIPTION
**Pull request by kommiharibabu**

### CODE REVIEW

File: src/reconcile/rec_visibility.c [hunk 1]
```diff
@@ -679,16 +679,27 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, W
...
-        if (upd->start_ts > tombstone->start_ts)
-            upd_select->ooo_tombstone = true;
+            while (upd != NULL && upd->txnid == WT_TXN_ABORTED)
+                upd = upd->next;
+
+            if (upd != NULL && upd->start_ts > tombstone->start_ts)
+                upd_select->ooo_tombstone = true;
 
-        if (vpack != NULL && vpack->tw.start_ts > upd->start_ts)
+            if (vpack != NULL && vpack->tw.start_ts > tombstone->start_ts)
```
Comment (etienneptl) [inline src/reconcile/rec_visibility.c:700]: Should this be `else if` or part of the first condition?
```
if ((upd != NULL && upd->start_ts > tombstone->start_ts) || (vpack != NULL && vpack->tw.start_ts > tombstone->start_ts))
  upd_select->ooo_tombstone = true;
```

Comment (kommiharibabu) [reply src/reconcile/rec_visibility.c]: Thanks. I rearranged the code as you suggested.

**Review by etienneptl — APPROVED** [review]
LGTM

**Review by clarissecheah — APPROVED** [review]
lgtm!

**Review by bvpvamsikrishna — APPROVED** [review]
lgtm

**Pull request merged**